src/display/general/overallGraphs.go

```diff
@@ -30,10 +29,52 @@ import (
 	"github.com/pesos/grofer/src/utils"
 )
 
+var myPage *MainPage
 var isCPUSet = false
-
 var run = true
 var helpVisible = false
+var cpuTableVisible = false
+
+func tableChange_3(selectedTable int) {
+	switch selectedTable {
+	case 0:
+		myPage.CPUTable.BorderStyle.Fg = ui.ColorClear
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorCyan
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.ShowCursor = false
+		myPage.TemperatureTable.ShowCursor = false
+
+	case 1:
+		myPage.CPUTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorClear
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.ShowCursor = true
+		myPage.TemperatureTable.ShowCursor = false
+
+	case 2:
+		myPage.CPUTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorCyan
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorClear
+		myPage.DiskChart.ShowCursor = false
+		myPage.TemperatureTable.ShowCursor = true
+	}
+}
+
+func tableChange_2(selectedTable int) {
+	switch selectedTable {
+	case 1:
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorClear
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.ShowCursor = true
+		myPage.TemperatureTable.ShowCursor = false
+
+	case 2:
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorCyan
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorClear
+		myPage.DiskChart.ShowCursor = false
+		myPage.TemperatureTable.ShowCursor = true
+	}
+}
```

Comment:
Could these be combined into a single function (with better name :P)? If needed you could pass `numCores` and `cpuTableVisible` to it.

Reply:
Done!